scriptworker/test/test_production.py

```diff
@@ -0,0 +1,187 @@
+    config = get_unfrozen_copy(DEFAULT_CONFIG)
+    del(config['credentials'])
+    if isinstance(override, dict):
+        config.update(override)
+    with open(os.path.join(basedir, "config.json"), "w") as fh:
+        json.dump(config, fh, indent=2, sort_keys=True)
+    config = apply_product_config(config)
+    return config
+
+
+@async_contextmanager
+async def get_context(config_override=None):
+    context = Context()
+    with tempfile.TemporaryDirectory() as tmp:
+        context.config = build_config(config_override, basedir=tmp)
+        credentials = read_integration_creds()
+        swlog.update_logging_config(context)
+        utils.cleanup(context)
+        async with aiohttp.ClientSession() as session:
+            context.session = session
+            context.credentials = credentials
+            yield context
+
+
+# verify_cot {{{1
+@pytest.mark.skipif(os.environ.get("NO_TESTS_OVER_WIRE"), reason=SKIP_REASON)
+@pytest.mark.parametrize("branch_context", ({
+    "name": "mozilla-central nightly desktop",
+    "index": "gecko.v2.mozilla-central.latest.firefox.decision-nightly-desktop",
+    "task_label_to_task_type": {
+        "balrog-.*-nightly/opt": "balrog",
+        "beetmover-repackage-.*-nightly/opt": "beetmover",
+    },
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-esr68 nightly android",
+    "index": "gecko.v2.mozilla-esr68.latest.firefox.decision-nightly-android",
+    "task_label_to_task_type": {
+        "push-apk/opt": "pushapk",
+    },
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-central nightly android",
+    "index": "gecko.v2.mozilla-central.latest.firefox.decision-nightly-android",
+    "task_label_to_task_type": {
+        "push-apk/opt": "pushapk",
+    },
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-central win64 en-US repackage signing",
+    "index": "gecko.v2.mozilla-central.nightly.latest.firefox.win64-nightly-repackage-signing",
+    "task_type": "signing",
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-beta win64 en-US repackage signing",
+    "index": "gecko.v2.mozilla-beta.nightly.latest.firefox.win64-nightly-repackage-signing",
+    "task_type": "signing",
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-release win64 en-US repackage signing",
+    "index": "gecko.v2.mozilla-release.nightly.latest.firefox.win64-nightly-repackage-signing",
+    "task_type": "signing",
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-esr60 win64 en-US repackage signing",
+    "index": "gecko.v2.mozilla-esr60.nightly.latest.firefox.win64-nightly-repackage-signing",
+    "task_type": "signing",
+    "cot_product": "firefox",
+}, {
+    "name": "mozilla-esr68 win64 en-US repackage signing",
+    "index": "gecko.v2.mozilla-esr60.nightly.latest.firefox.win64-nightly-repackage-signing",
+    "task_type": "signing",
+    "cot_product": "firefox",
+}, {
+    "name": "fenix nightly",
+    "index": "project.mobile.fenix.v2.nightly.latest",
```

Comment:
Awesome!
If we're able to verify a later task, e.g. a signing task or pushapk task, even better. That may be dependent on what's indexed.

Reply:
:+1: Signing tasks are the ones indexed at `project.mobile.fenix.v2.nightly.latest` (for instance https://tools.taskcluster.net/groups/ahv5e-kfQUCtF-2J87nUHw/tasks/b3T1dbdJQMGqnaVpBBAS1Q/details)

IIRC, we haven't indexed pushapk tasks. Doing so shouldn't be a problem though.